recipes/default.rb

```diff
@@ -55,3 +55,9 @@
     key          node['graylog2']['server']['repos']['debian']['key']
   end
 end
+
+build_essential '' do
+  compile_time true
+end
+
+chef_gem 'mongo'
```

Comment:
Same here, not everybody wants to use the token feature. This should be in a separate recipe so that people can decide if they want to pull in this dependency.